spec/unit/remote_system_spec.rb

```diff
@@ -225,6 +225,16 @@
     end
 
     describe "#read_file" do
+      it "allows retrieving files without privileged mode by default" do
```

Comment:
That reads a bit weird, what about `retrieves files in unprivileged mode by default`?